cli/src/factory_ops/unboxing/config_update

```diff
@@ -224,6 +224,215 @@ function update_cluster_sls {
+    echo "Checking if the  RedHat url is removed from commons pillar" >> ${LOG_FILE}
+
+    # Check if RedHat url is removed from commons url on both nodes
+    _node1_url=$(salt srvnode-1 pillar.get commons:cortx_commons:RedHat --output=newline_values_only ${salt_opts})
+    _node2_url=$(salt srvnode-2 pillar.get commons:cortx_commons:RedHat --output=newline_values_only ${salt_opts})
+    if [[ ! -z "$_node1_url" ]]; then
+        echo "ERROR: RedHat url could not be removed from commons url on srvnode-1" | tee -a ${LOG_FILE}
+        exit 1
+    fi
+
+    if [[ ! -z "$_node2_url" ]]; then
+        echo "ERROR: RedHat url could not be removed from commons url on srvnode-2" | tee -a ${LOG_FILE}
+        exit 1
+    fi
+
+    echo "RedHat url is removed from commons pillar" >> ${LOG_FILE}
+
+    # Check if CentOS url is removed from commons url on both nodes
+    _node1_url=$(salt srvnode-1 pillar.get commons:cortx_commons:CentOS --output=newline_values_only ${salt_opts})
+    _node2_url=$(salt srvnode-2 pillar.get commons:cortx_commons:CentOS --output=newline_values_only ${salt_opts})
+    if [[ ! -z "$_node1_url" ]]; then
+        echo "ERROR: CentOS url could not be removed from commons url on Server A" | tee -a ${LOG_FILE}
+        exit 1
+    fi
+
+    if [[ ! -z "$_node2_url" ]]; then
+        echo "ERROR: CentOS url could not be removed from commons url on Server B" | tee -a ${LOG_FILE}
+        exit 1
+    fi
+
+    echo "CentOS url is removed from commons pillar" >> ${LOG_FILE}
+
+    echo "Done" | tee -a ${LOG_FILE}
+}
+
+function remove_target_build {
+
+    echo "Removing CORTX build url from configuration" | tee -a ${LOG_FILE}
+    local _release_sls_path=${PRVSNR_ROOT}/pillar/components/release.sls
+    yes | cp -f $_release_sls_path ${_release_sls_path}.bkp
+    local _tgt_build=$(grep -n "target_build:"  $_release_sls_path | awk '{ print $3 }')
+    if [[ ! -z $_tgt_build ]]; then
+        line_target_build=`grep -n "target_build:"  $_release_sls_path | cut -d: -f1`
+        if [[ ! -z $line_target_build ]]; then
+            echo "Removing target_build from $_release_sls_path" >> $LOG_FILE
+            sed -ie "${line_target_build}s/.*/    target_build:/" $_release_sls_path
+            n_diff_lines=`diff $_release_sls_path ${_release_sls_path}.bkp | grep "^>" | wc -l`
+            echo "DEBUG: n_diff_lines: $n_diff_lines" >>  ${LOG_FILE}
+            if [[ $n_diff_lines != 1 ]]; then
+                echo "Could not update the target_build in $_release_sls_path, restoring the original content" | tee -a ${LOG_FILE}
+                yes | cp -f ${_release_sls_path}.bkp $_release_sls_path
+            else
+                echo "Successfully updated target_build in $_release_sls_path" >> ${LOG_FILE}
+            fi
+        else
+            echo "Could not find the target_build line number in $_release_sls_path, ignoring" >> $LOG_FILE
+        fi
+    fi
+
+    if [[ -f "${PRVSNR_ROOT}/pillar/user/groups/all/release.sls" ]]; then
```

Comment:
this path won't work after new deployment method.

Reply:
addressed